src/_help/publish-documentation/deploy-and-release-management.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Deploy and release management
+---
+
+- TOC
+{:toc}
+
+## What are deployments
+
+Deployment corresponds to the publication of an uploaded API definition of documentation.
+
+We call an "API document" the file that contains your API definition, the content of your API. When you upload and then deploy an API document, it creates or updates an API version: a marked and named iteration of your API.
+
+When your API undergoes a structural change, it is reflected on Bump.sh in the changelog. This change is referred to as an "API change."
+
+![](/images/help/deployments-list.png)
+
+By default, an API document is automatically deployed after being uploaded. However, you can choose to deploy it manually at a later time by using the Manual Release mode. In this case, the latest deployments will be enqueued until you decide to release one of them.
+
+You can also add context (title, description) to any release afterward. This option is available from the page of the deployment, accessible in the list of deployments for your documentation.
```

Comment:
I reworked this part to make it easier to understand (and fixed the mistakes). WDYT of this proposal?

```suggestion
A Deployment is the processing of an API document - containing your API definition - sent to Bump.sh servers. 

Following this deployment, the new API definition is published to your API consumers. This is what we call a Release: your API documentation is updated, and if your API has changed since the last deployment, a new entry representing this API change is added to your API changelog.

![](/images/help/deployments-list.png)

By default, a deployment is automatically released after its processing. However, you can choose to control the release flow by activating the Manual Release setting. In this case, the latest deployments will be enqueued until you decide to release one of them.

You can also add context (title, description) to any release afterward. This option is available from the page of the deployment, accessible in the list of deployments for your documentation.
```

(Do not hesitate to fix my English if needed, it's more about simplifying the concepts that working on sentences)